tests/contrib/test_viz/tail_probs_hypotest_results.json

```diff
@@ -0,0 +1 @@
83e-08, 0.44052488676537493], [3.581843984973475e-12, 5.724240363635464e-10, 7.611004895914735e-08, 7.179762309886526e-06, 0.00037614276876065896]], [1.2204302842607375e-08, [5.375556234151579e-09, 0.44046401531307167], [8.289016229526988e-13, 1.6002294343557796e-10, 2.56735659778783e-08, 2.917351596862768e-06, 0.0001835965573621605]], [3.894067965057213e-09, [1.715006793048495e-09, 0.4404152183366675], [1.871672851436178e-13, 4.359198972596319e-11, 8.428668544601014e-09, 1.1524729491677362e-06, 8.705487289700728e-05]], [1.2104006975875942e-09, [5.330336083189006e-10, 0.4403778099114372], [4.126369816628468e-14, 1.1579097656551975e-11, 2.694900489329346e-09, 4.42904105646278e-07, 4.0122566933144385e-05]], [3.667493739627566e-10, [1.6149850132598882e-10, 0.44035113020367145], [8.887718335956289e-15, 3.000975712785147e-12, 8.396794954296739e-10, 1.656895757074108e-07, 1.7984640006926466e-05]], [1.0839184236026052e-10, [4.7728672736408496e-11, 0.4403345463745634], [1.871376477833483e-15, 7.593443691503971e-13, 2.5511797690607427e-10, 6.037428992972254e-08, 7.844792431947736e-06]], [3.1266276203322805e-11, [1.376739975779294e-11, 0.44032745275658436], [3.8542324194372824e-16, 1.8770035561845096e-13, 7.562900841117337e-11, 2.1440757204129407e-08, 3.331770799781985e-06]], [8.807777916102956e-12, [3.878322427997221e-12, 0.4403292708943783], [7.769087418870065e-17, 4.5351922804897234e-14, 2.1888306693239685e-11, 7.425300739683435e-09, 1.378565269350356e-06]], [2.4244792489265334e-12, [1.0675938572918763e-12, 0.44033944929186997], [1.533555829124501e-17, 1.0717111527033957e-14, 6.188172059269856e-12, 2.509129073317249e-09, 5.560059675205046e-07]], [6.524945211405893e-13, [2.8733083197921885e-13, 0.44035746304344725], [2.965934025355457e-18, 2.4782856816222296e-15, 1.7099467199930306e-12, 8.277717406724008e-10, 2.1871024270051275e-07]], [1.71782565371694e-13, [7.565008939636483e-14, 0.4403828131957232], [5.623198801534755e-19, 5.611123291797592e-16, 4.620721835800917e-13, 2.667563671013248e-10, 8.395178386244305e-08]], [4.4264538053488135e-14, [1.9494767682506594e-14, 0.4404150261084757], [1.0456498253680896e-19, 1.2445134378447365e-16, 1.2217287655073372e-13, 8.401713598966383e-11, 3.1462347681656477e-08]], [1.1169446644370348e-14, [4.9196235733322765e-15, 0.4404536527162585], [1.9080250006767082e-20, 2.7053322683717073e-17, 3.162291370596417e-14, 2.587595613009092e-11, 1.1518009445150847e-08]], [2.7613663488989174e-15, [1.2163770933924125e-15, 0.44049826777871665], [3.418096559152543e-21, 5.766686417109634e-18, 8.016951593246653e-15, 7.796887738082306e-12, 4.121059344016396e-09]], [6.691824089371058e-16, [2.9480728583174304e-16, 0.4405484691386306], [6.014343036613147e-22, 1.2059350559181132e-18, 1.991631729977912e-15, 2.299628789191756e-12, 1.4417855671818827e-09]], [1.590370785130139e-16, [7.007235252659966e-17, 0.44060387163654846], [1.0398976286066765e-22, 2.4752233266368097e-19, 4.850732788241502e-16, 6.642253139501716e-13, 4.934728037238966e-10]]]}
```

Comment:
this I guess is not yet the format @alexander-held designed for µ-scans?

Reply:
> this I guess is not yet the format/schema @alexander-held designed for µ-scans?

Sorry, can you link me (again maybe?)? I think I missed this.